compiler/data/performance-inspections.h

```diff
@@ -0,0 +1,50 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2020 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+#pragma once
+#include "compiler/data/data_ptr.h"
+#include "common/wrappers/string_view.h"
+
+
+class PerformanceInspections {
+public:
+  enum Inspections : uint8_t {
```

Comment:
I don't use enum class because this enum is inside `PerformanceInspections` class, so it is required to use this class namespace `PerformanceInspections::`, and `PerformanceInspections::Inspections::` looks wired for me.